include/fastrtps/serializer/serializer.h

```diff
@@ -0,0 +1,502 @@
+#include <fastrtps/types/DynamicTypePtr.h>
+#include <fastrtps/types/TypeObjectHashId.h>
+#include <fastrtps/types/TypeObject.h>
+namespace eprosima{
+
+namespace fastcdr{
+    class Cdr ;
+}
+namespace fastrtps{
+
+namespace types{
+    class DynamicData ;
+}
+
+namespace serializer{
+
+/*********************
+ *     SERIALIZER    *
+ *********************/
+	
+template<class Serializable>
+class Serializer{
+private:
+    Serializer() = delete ;
+public:
+    Serializer<Serializable>(Serializable *p):p_(p){}
+    
+    void serialize(eprosima::fastcdr::Cdr& cdr)
+				{
+        serialize(p_, cdr) ;
+    }
+    void deserialize(eprosima::fastcdr::Cdr& cdr)
```

Comment:
Coding style:
Methods must be separated by a blank line.

Reply:
done